client/ayon_blender/api/ops.py

```diff
@@ -378,6 +379,18 @@ def execute(self, context):
         pipeline.set_resolution(data)
         return {"FINISHED"}
 
+class SetUnitScale(bpy.types.Operator):
+    bl_idname = "wm.ayon_set_unit_scale"
+    bl_label = "Set Unit Scale"
+
+    def execute(self, context):
+        project = os.environ.get("AYON_PROJECT_NAME")
```

Comment:
Getting the current project name should not rely solely on `os.environ`. Don't we have dedicated functions to "get the current project name"? We should rely on those.

I believe there's even a `get_current_project_settings` function somewhere to import as well.

Reply:
This is marked resolved, but code still seems old?